toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/IapAuthenticator.kt

```diff
@@ -38,7 +38,7 @@ internal fun IapSignInAuthenticator(
     onComplete : (String) -> Unit,
     onCancel: () -> Unit,
 ) {
-    var hasLaunched by rememberSaveable { mutableStateOf(false) }
+    var hasLaunched by rememberSaveable(authorizedUrl) { mutableStateOf(false) }
```

Comment:
reminder to change this parameter to 

```suggestion
    var hasLaunched by rememberSaveable(authorizeUrl) { mutableStateOf(false) }
```

(now that you touched it in this PR :) )